Makefile

```diff
@@ -0,0 +1,19 @@
+.PHONY: commit lint install check
+
+xargs=$(if $(shell echo | xargs -r 2>/dev/null && echo 1), xargs -r, xargs)
+
+GIT_DIFF=git diff --name-only --cached --diff-filter=dt
+commit:
+	${GIT_DIFF} -- '*.py' | $(xargs) isort --check-only --diff
+	${GIT_DIFF} -- '*.py' | $(xargs) flake8
```

Comment:
`${GIT_DIFF}` -> `$(git_diff)`, for consistency with `$(xargs)`

Reply:
`GIT_DIFF` more like constant, while `$(xargs)` is command.

Reply:
Renamed in #3